packages/lib/server-only/user/generate-confirmation-token.ts

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+
+import { prisma } from '@documenso/prisma';
+
+import { ONE_HOUR } from '../../constants/time';
+import { sendConfirmationEmail } from '../auth/send-confirmation-email';
+
+const IDENTIFIER = 'confirmation-email';
+
+export const generateConfirmationToken = async ({ email }: { email: string }) => {
+  const now = new Date();
+  const token = crypto.randomBytes(20).toString('hex');
+
+  const expirationDate = new Date(now.getTime() + ONE_HOUR);
```

Comment:
Normally we do `Date.now() + ONE_HOUR` I can't see us reusing the date object below either so we should probably keep it simple 😄